packages/plugin-ext/src/main/browser/plugin-frontend-contribution.ts

```diff
@@ -33,6 +37,16 @@ export class PluginApiFrontendContribution implements CommandContribution {
     @inject(OpenUriCommandHandler)
     protected readonly openUriCommandHandler: OpenUriCommandHandler;
 
+    @inject(ApplicationShell)
+    protected readonly shell: ApplicationShell;
+
+    @inject(EditorManager)
+    protected readonly editors: EditorManager;
+
+    @inject(OpenerService)
+    protected readonly openerService: OpenerService;
+
+
```

Comment:
travis-ci doesn't like this consecutive blank line

Reply:
our linter sometimes is very annoying :)

Reply:
we could add tslint --fix script that fixes automatically tslint errors

Reply:
let's discuss next week on dev meeting? i guess it should be git hook or something like that

Reply:
you can discuss it but I'm on PTO next week and at RHT summit the week after so I may not be able to attend the dev meeting.